CHANGELOG.md

```diff
@@ -119,7 +119,7 @@ All notable changes to this project will be documented in this file. For future
 - Added tooling to prevent changelog conflicts.
 - Added warning when configuring m2n as mpi when preCICE is compiled with OpenMPI. This is known to cause connection issues.
 - Allowed multiple convergence measures of same coupling data.
-- Changed `com::MPIDirectcommunication` to work only for Master-Slave connections.
+- Changed `com::MPIDirectcommunication` to work only for intra-participant communication connections.
```

Comment:
This, I would not change. In 2.1, we still used the "old" terminology.

Reply:
Good catch! CHANGELOG should definitely not change.